net/bridge.go

```diff
@@ -530,11 +558,17 @@ func configureIPTables(config *BridgeConfig, ips ipset.Interface) error {
 	// src IP addr.
 	if config.NoMasqLocal {
 		ips := ipset.New(common.LogLogger(), 0)
-		_ = ips.Destroy(NoMasqLocalIpset)
-		if err := ips.Create(NoMasqLocalIpset, ipset.HashNet); err != nil {
-			return err
+
+		noMasqLocalExists, err := ips.Exists(NoMasqLocalIpset)
+		if err != nil {
+			common.Log.Errorf("Failed to look if ipset '%s' exists: %s", NoMasqLocalIpset, err)
+		} else if !noMasqLocalExists {
+			if err := ips.Create(NoMasqLocalIpset, ipset.HashNet); err != nil {
+				return err
+			}
```

Comment:
On reflection I think this change is wrong: we don't make the tracker restart from scratch so if we do recreate the ipset mid-flight it won't have the right stuff in it.  I think it is ok just to move the destroy/recreate to `ResetIPTables()` and assume it exists here.